client/state/preview/reducer.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import assign from 'lodash/assign';
+
+/**
+ * Internal dependencies
+ */
+import * as ActionTypes from 'state/action-types';
```

Comment:
One thing to note is that when we move to a tree-shaking build tool, e.g. webpack2, `import *` won't be optimised. This doesn't matter for now, and won't matter for this particular import — it's used globally and is tiny — but it's something to bear in mind anyway.